Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4210,18 +4210,18 @@ ObservableList<ElementLocator> IWindowExplorer.GetElementLocators(ElementInfo El
             string id = e.GetAttribute("id");
             if (!string.IsNullOrEmpty(id))
             {
-                list.Add(new ElementLocator() { LocateBy = eLocateBy.ByID, LocateValue = id, Help = "Very Recommended (usually unique)", Active = true });
+                list.Add(new ElementLocator() { LocateBy = eLocateBy.ByID, LocateValue = id, Help = "Very Recommended (usually unique)", Active = true, IsAutoLearned = true });
             }
             string name = e.GetAttribute("name");
             if (!string.IsNullOrEmpty(name))
             {
                 list.Add(new ElementLocator() { LocateBy = eLocateBy.ByName, LocateValue = name, Help = "Very Recommended (usually unique)", Active = true });
             }
```

Comment:
you missed one- eLocateBy.ByName, please add IsAutoLearned = true to it

Reply:
Added